lib/Config.js

```diff
@@ -39,6 +41,9 @@ export function locationConstraintAssert(locationConstraints) {
         assert(typeof locationConstraints[l].type === 'string',
             'bad config: locationConstraints[region].type is ' +
             'mandatory and must be a string');
+        assert(supportedBackends.indexOf(locationConstraints[l].type) > -1,
```

Comment:
I had encountered this before: the `includes` array method did not work with our previous node version 4. However it should be good now that we're using node v6.9.5.